R/cranPackage.R

```diff
@@ -0,0 +1,20 @@
+#' getCranPackage
+#'
+#' get CranPackage object from pkgPath. Returns NULL if there is no CRAN package.
+#'
+#' @param pkgPath package path
+#'
+#' @export
+#'
+#' @return CranPackage or NULL if it does not exist
+getCranPackage <- function(pkgPath) {
+  result <- NULL
+  pkgName <- basename(pkgPath)
+  tryCatch({
+    result <- pkginfo::CranPackage$new(pkgName)
```

Comment:
Could you see if you can achieve the same thing with [`pak`](https://cran.r-project.org/web/packages/pak/index.html).